src/app.ts

```diff
@@ -21,7 +21,10 @@ import {TestRuns} from "./test-runs";
 // TestRuns.testPreProcessingChain();
 
 //Testing of WetManager
-TestRuns.testWetManager();
+//TestRuns.testWetManager();
```

Comment:
`TestRuns` were never intended to do unit tests. It is just a playground to run different combinations of modules. Unit tests are placed in the `tests` folder (is currently refactored by @lgfrbcsgo anyway).